src/engine/Frame.java

```diff
@@ -32,7 +32,7 @@ public class Frame extends JFrame {
 	 */
 	public Frame(final int width, final int height) {
 		setSize(width, height);
-		setResizable(false);
```

Comment:
I think this change is unnecessary.
Because, you created one more frame for 2p.